### ISSUE TITLE
nonexistent audio in the book's videos

### ISSUE THREAD

**Issue by irunestone** [RunestoneInteractive/RunestoneComponents#518]
Error reported in course pythonds on page Dictionaries by user lorenzo lorenzo.dragani@gmail.com
After completing almost two chapters of the book I have to say that I haven't been able to hear the audio of even one of the videos so far, despite having tried with two different computers and a tablet (two Apple computers with latest version of Firefox and Android table with latest version of Chrome).

**Comment by bnmnetp** [comment]
You are correct, Firefox seems to have the sound muted permanently for the videos.  It may have something to do with the videos being served over a non https connection.  Chrome and Safari on the mac are working fine however.

**Comment by luke0b1000** [comment]
I can confirm this happens on Windows / Firefox 58.0.2, gives your no way to unmute, even though right-clicking says unmuted, no sound comes through.

**Comment by bnmnetp** [comment]
I am working on migrating all of the videos to YouTube where they can be served via https.  But that is going to take a bit of time as I need to upgrade the youtube directive to handle our analytics recording properly.
